Michael Bustillos- Advanced

```diff
@@ -0,0 +1,58 @@
+// Exercise 4: Advanced JavaScript Practice
+// In this exercise, you will have practice using arrow functions
+// and class creation
+
+// Problem 1: Division object
+// Create the divide functionality in the following object
+// the divide function will return a new array with the nums
+// divided by the divideBy property.
+// Look into the map function to help :)
+const divider = {
+    nums: [100, 200, 300, 400, 500],
+    divideBy: 10,
+    // create divide function here
+    divide() {
+        doIt = () => {
+            for(let i = 0; i < this.nums.length; i++) {
```

Comment:
I think I was unclear so my bad, but use map to divide each element in nums by divideBy and return that array!